typing/test/Assertation.specs.ts

```diff
@@ -0,0 +1,30 @@
+import {expect, use} from 'chai';

```

Comment:
Why are the tests in 2 different files instead of just `chai-immutable.specs.ts`?